### PR TITLE
Python cluster loader apiserver restarts

### PR DESCRIPTION
**Pull request by akrzos**
Use logging and oc command with retries for MasterVertical

For MasterVertical test, we need to retry commands in event the apiserver is rebuilt during a MasterVertical run which is likely due to the certificate rotation time and length of mastervertical test.  To support debugging this, I also added logging to capture stdout/stderr of the oc commands and converted print statements to use the logger.  This should expose other places we may need to implement the retry mechanism.

### CODE REVIEW

**Comment by akrzos** [comment]
@mffiedler @chaitanyaenr PTAL

File: openshift_scalability/config/pyconfigMasterVertScalePause.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 projects:
   - num: 3
     basename: c
+    node_selector_label: "computenode=true"
```
Comment (chaitanyaenr) [inline openshift_scalability/config/pyconfigMasterVertScalePause.yaml]: Now that the PR to allow empty node selector got merged, we can set this to standard worker node label.

Comment (sjug) [reply openshift_scalability/config/pyconfigMasterVertScalePause.yaml]: Can we not use this `node_selector_label` name?

**Comment by akrzos** [comment]
@mffiedler Did you want to take a look so we can get this merged and incase there are other tests using the python cluster loader?

**Comment by mffiedler** [comment]
@akrzos Have been trying to find time to test this.   Will try to get to it by early next week.

@hongkailiu @wabouhamad Can you think of any automation that the additional logger output might affect.  I think we're only checking oc output.

**Comment by hongkailiu** [comment]
@mffiedler I have used this version when testing OCP 4 with scale lab because without retry many tests would not even work.
Only one thing ... with this PR merged, we would make `oc-cli` failures (potential bugs) more invisible unless we check the retry logs which is `sys.stdout` for the moment.

Back to your question: No, I cannot think of any impact by more logging entries.

**Comment by wabouhamad** [comment]
@mffiedler I think we'll be good automation wise with the logger output, I don't see where it could cause issues.  LGTM

**Pull request merged**